src/googleMusic.ts

```diff
@@ -54,9 +65,26 @@ export default class GoogleMusic {
     // Receiving data from GPMDP.
     this.ws.on('message', data => {
       let gMusicResponse: GoogleMusicResponse = JSON.parse(data.toString())
-      switch (gMusicResponse.channel) {
+      let gMusicResult: GoogleMusicResultResponse = gMusicResponse as GoogleMusicResultResponse
+      if (gMusicResult.namespace) {
+        switch (gMusicResult.requestID) {
+          case 'extra.getTrackURL':
+            window
+              .showInformationMessage(
+                `${this._track.title} - ${this._track.artist} \n ${gMusicResult.value}`,
+                'Copy URL to clipboard'
+              )
+              .then(() => {
```

Comment:
This still copys the URL even if the user clicks (X).
Instead of requesting user input, just copy the URL to the clipboard then let the user know the operation was successful.